modulelogger/module_logger_test.go

```diff
@@ -0,0 +1,257 @@
+package modulelogger_test // avoid cycles
+
+import (
+	"github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/suite"
+	"github.com/uber-common/bark"
+	"github.com/uber/ringpop-go/modulelogger"
+	"github.com/uber/ringpop-go/test/mocks"
+	"testing"
+)
+
+// Just like mocks.Logger but with convenient On and Onf methods.
+type mockLogger struct {
+	*mocks.Logger
+}
+
+// mock.On("Debug", "msg") instead of mock.On("Debug", []interface{}{"msg"})
+func (mock *mockLogger) On(methodName string, arguments ...interface{}) *mock.Call {
+	return mock.Logger.On(methodName, arguments)
+}
+
+func (mock *mockLogger) Onf(methodName string, format string, arguments ...interface{}) *mock.Call {
+	return mock.Logger.On(methodName, format, arguments)
+}
+
+func newMockLogger() *mockLogger {
+	return &mockLogger{Logger: &mocks.Logger{}}
+}
+
+type ModuleLoggerSuite struct {
+	suite.Suite
+	// unit under test
+	uut *modulelogger.ModuleLogger
```

Comment:
can you rename `uut` to something like `logger`? abbreviations/acronyms are hard to remember. `uut` is uncommon as far as I'm aware.